yarn.lock

```diff
@@ -7984,6 +8014,17 @@ markdown-it@^11.0.0:
     mdurl "^1.0.1"
     uc.micro "^1.0.5"
 
+markdown-it@^12.3.2:
```

Comment:
Interesting this just added an entry to `yarn.lock` 🤔 I wonder if it would be beneficial to remove all `markdown-it` entries and re-run `yarn` again?

Reply:
I'm pretty sure that's what I tried first. I double check though.

Reply:
Fixed both issues.